style/js/README.md

```diff
@@ -1,3 +1,163 @@
+JavaScript Style Guides
+=====
+
+Guides for programing JavaScript in style
+
+General
+---
+* Use [ESLint](http://eslint.org/)
+* Always use `use strict`
+
+Parens and Braces
+---
+* Leave one space between `if` and `(`.
+
+```javascript
+if (something) {
+  //some work here;
+}
+```
+
+* Prefer this form for multiline if statements.
+
+```javascript
+if (something) {
+  // do stuff
+  // even more stuff
+} else {
+  // some other stuff
+  // real stuff
+}
+```
+
+* Don't include spaces before `(` and function declaration.
+
+```javascript
+function bar() {
+  return  foo;
+}
+
+var bar = function() {
+  return foo;
+}
+```
+
+Assignments and Declarations
+---
+* Initialize arrays with `[ ]`.
+
+```javascript
+var array = [];
+```
+
+* Initialize objects with `{ }`.
+
+```javascript
+var object = {};
+```
+
+* Leave one space after `{` and before `}` when initializing objects on the same line.
+
+```javascript
+var map = { ready: 9, when: 4, "you are": 15 };
+```
+
+* Prefer single line object/array initialization.
```

Comment:
You can ask the question the other way around, do we want the other form on our repos? Let the other developers pitch in too so we can understand what is really prefered by the majority instead of guessing around :walking: